settings_page.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+// A class to create and manage the admin's B2C settings page
+class settings_page
```

Comment:
If you want to be very correct, per [https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions):

 - The file should be named `class-settings-page.php`
 - The class should be named `Settings_Page`

To avoid class name conflict with another plugin ("Settings_Page" is pretty common), it would be even better to prepend the class name with the slug that represents the plugin (though I don't think this one has one yet). So if the plugin's slug were "AADB2C", you would name the file `class-aadb2c-settings-page.php`, and the class would be `AADB2C_Settings_Page`.

---
In reply to: [71194133](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71194133) [](ancestors = 71194133)